### PR TITLE
Update FSF address

### PR DESCRIPTION
**Pull request by tamplan**
Update [FSF address](https://www.fsf.org/about/contact/)

### CODE REVIEW

File: battstat/power-management.c [hunk 1]
```diff
@@ -14,7 +14,7 @@
  *
  *  You should have received a copy of the GNU General Public License
  *  along with this program; if not, write to the Free Software
- *  Foundation, Inc., 59 Temple Street #330, Boston, MA 02110-1301, USA.
+ *  Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
```
Comment (rbuj) [inline battstat/power-management.c]: `Free Software` already exists on the end of the previous line.

Comment (tamplan) [reply battstat/power-management.c]: Yes, seen after create PR, correct now, the others too.

Thanks

**Comment by raveit65** [comment]
Beside from missing comma after `MA 02110-1301` it looks Ok.

**Comment by rbuj** [comment]
```shell
make distclean
find . -type f -name "*.c" -print0 | xargs -0 sed -i 's/Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA./Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA./g'
find . -type f -name "*.h" -print0 | xargs -0 sed -i 's/Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA./Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA./g'
find . -name *.c -exec git add {} +
find . -name *.h -exec git add {} +
git commit --amend
git push -f
```

**Comment by tamplan** [comment]
> Beside from missing comma after `MA 02110-1301` it looks Ok.

A little mistake, "Free Software" is twice (a line before at the end).
I'll update my commits...

**Comment by tamplan** [comment]
@rbuj, thanks you very much, done!

**Pull request merged**